src/internal/packager/images/common.go

```diff
@@ -107,9 +107,9 @@ func createPushOpts(cfg PushConfig, pb *message.ProgressBar) []crane.Option {
 	// TODO (@WSTARR) This is set to match the TLSHandshakeTimeout to potentially mitigate effects of https://github.com/zarf-dev/zarf/issues/1444
 	transport.ResponseHeaderTimeout = 10 * time.Second
 
-	transportWithProgressBar := helpers.NewTransport(transport, pb)
+	transportWithoutProgressBar := helpers.NewTransport(transport, nil)
```

Comment:
The helpers.NewTransport is just [oras.NewTransport](https://github.com/oras-project/oras-go/blob/main/registry/remote/retry/client.go#L45) with a progressbar. Should we switch to that instead?

Reply:
I'm trying to keep the change minimal rn to see if it helps with the issue, however if this stays longterm we should do that